### PR TITLE
Added a section about communicating task progress among team members.

### PR DESCRIPTION
**Pull request by alexma22**
- Added a section describing the importance of understanding project progress
- Included examples of how it would help during product development
- Included examples of software to use that can help in this area

### CODE REVIEW

File: Topics/Teamwork.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ Working as a team effectively requires a lot of planning and organization. Proje
 ## Team Communication
 During the early stages of forming a team, the group should also establish general expectations for how the team plans to communicate with one another. This may include setting up a team communication channel on a specific platform (ex. discord or slack) and establishing regular check-up meetings (including when these meetings are and where they will be held - for example, in person or online). These general meetings can be used to outline general expectations and project requirements, update one another on individual progress, delegate new reponsibilities, and set project deadlines.
```
Comment (LiamOdero) [inline Topics/Teamwork.md:13]: A sub-header would help visually signify the topic. Looking at the above header, I would suggest using `###`

**Review by LiamOdero — CHANGES_REQUESTED** [review]
Overall, this looks like a great excerpt on tracking task progress, though there are a few tweaks that I think could enhance it

File: Topics/Teamwork.md (hunk 1 above)
Comment (LiamOdero) [inline Topics/Teamwork.md:13]: A sub-header here would help signify the topic. I'd suggest using `###` going by the above header

File: Topics/Teamwork.md [hunk 2]
```diff
@@ -11,6 +11,14 @@ Working as a team effectively requires a lot of planning and organization. Proje
...
+
+In general, when setting goals for your team as a project manager, their are many things to consider, such as the clarity of the goal, the fact that it is a realistic goal, and that the goal is relevant and important to your project. Proper communication with your team members will aid in acheiving each of these goals, and correctly following these tips will help your team reach their goals and provide a better product to your client. In this process, organization is key, and a project manager must try to leverage all the tools available to him to improve his ability to manage the team. When it comes to deadline-setting and task prioritizing, one of the most powerful tools that a manager can use is a todo list or project board/roadmap.
+
+There are many useful options for this, such as [Notion](https://notion.notion.site/Product-roadmap-c5e8829bb9644dd08c576452ee200404) and [Trello](https://trello.com/b/1x4Uql2u/project-management. Both of these tools allow a project manager and their team to create a page, that they can all access to check important tasks and notes about those tasks.
```
Comment (LiamOdero) [inline Topics/Teamwork.md:20]: Images of notion and/or trello being used to track progress with brief explanations would enhance this section

File: Topics/Teamwork.md [hunk 3]
```diff
@@ -11,6 +11,14 @@ Working as a team effectively requires a lot of planning and organization. Proje
 ## Team Communication
 During the early stages of forming a team, the group should also establish general expectations for how the team plans to communicate with one another. This may include setting up a team communication channel on a specific platform (ex. discord or slack) and establishing regular check-up meetings (including when these meetings are and where they will be held - for example, in person or online). These general meetings can be used to outline general expectations and project requirements, update one another on individual progress, delegate new reponsibilities, and set project deadlines. 
 
+An important part of team communication is in communicating tasks, deadlines and progress. Each team member should have good knowledge of the general progress of the project. This will give each team member a deeper understanding of the current state of the project so that they can more adequetly allocate their time. Perhaps even more importantly, the project manager will be able to have a deeper understanding of the progress of the project, and the team member's working habits and they will be able to manage the team more adequetly.
```
Comment (LiamOdero) [inline Topics/Teamwork.md:14]: A sub-header at the start of the section you added would help signify it. Looking at the above header, I'd suggest using `###`

File: Topics/Teamwork.md [hunk 4]
```diff
@@ -11,6 +11,14 @@ Working as a team effectively requires a lot of planning and organization. Proje
...
+
+In general, when setting goals for your team as a project manager, their are many things to consider, such as the clarity of the goal, the fact that it is a realistic goal, and that the goal is relevant and important to your project. Proper communication with your team members will aid in acheiving each of these goals, and correctly following these tips will help your team reach their goals and provide a better product to your client. In this process, organization is key, and a project manager must try to leverage all the tools available to him to improve his ability to manage the team. When it comes to deadline-setting and task prioritizing, one of the most powerful tools that a manager can use is a todo list or project board/roadmap.
```
Comment (LiamOdero) [inline Topics/Teamwork.md:18]: 'their' should be they're, though this is a minor grammatical error

File: Topics/Teamwork.md (hunk 3 above)
Comment (alexma22) [reply Topics/Teamwork.md:14]: Thanks for the suggestion, is there any titles that you think would work best here?

File: Topics/Teamwork.md (hunk 4 above)
Comment (alexma22) [reply Topics/Teamwork.md:18]: I think you meant 'there' but thanks for pointing out the issue.

File: Topics/Teamwork.md (hunk 2 above)
Comment (alexma22) [reply Topics/Teamwork.md:20]: Thanks for the idea, I'll go ahead and add some.

File: Topics/Teamwork.md (hunk 3 above)
Comment (LiamOdero) [reply Topics/Teamwork.md:14]: Time management might be an appropriate sub-title to use

Comment (alexma22) [reply Topics/Teamwork.md:14]: Thanks for the suggestion, that sounds like a good name.

**Review by LiamOdero — APPROVED** [review]
Article looks good to me now, nice job!

**Pull request merged**